### DISCUSSION TITLE
Redis version 6.2 - connection establishment

### DISCUSSION THREAD

**Discussion by Sanjay-Ashwin-Muthaiya** [NodeRedis/node_redis#2445] [Q&A]
I'm trying to upgrade the redis client version to 4.2.0 from 3.1.2. I could see as a breaking change we need to establish connect() to create connection. But without modinfing those connection change while upgrading the version, the changes are working without any issues. Could some share any insights on this ?

**Comment by leibale** [comment]
I'm pretty sure that the `connect` function existed in v3, just wasn't mandatory.. So I guess your code used `client.connect` with v3 as well..
